cico_run_tests.sh

```diff
@@ -0,0 +1,19 @@
+
+#!/bin/bash
+
+. cico_setup.sh
+
+TAG="latest"
+TEST_IMAGE_NAME="supervisor-tests:${TAG}"
```

Comment:
Can this be along the lines of https://github.com/fabric8-hdd/supervisor/pull/40/files#diff-b67911656ef5d18c4ae36cb6741b7965L23 and https://github.com/fabric8-hdd/supervisor/blob/master/cico_build_deploy.sh#L9

Reply:
so that the single point of update for this kind of things would then be the makefile